sap/cli/startrfc.py

```diff
@@ -97,6 +104,22 @@ def startrfc(connection, args):
         return 1
 
     # rfc call passed, it is time for analysis of the returned response
+    response_formatted = FORMATTERS[args.output](resp)
+
+    # if the response file is given
+    if args.response_file:
+        try:
+            # try to dump the formatted response into a completely new file
+            with open(args.response_file, 'x', encoding='utf-8') as file_obj:
+                file_obj.write(response_formatted)
+        except FileExistsError:
```

Comment:
I guess this case cannot happen, you are checking existnce of the file above. But it is always better to have more checks than none.

Reply:
Have you heard of race-conditions already? :)

Reply:
No ... :-) Ok, makes sense, I forgot there can be more instances with same params running in parallel.